utils/swift_build_support/swift_build_support/products/indexstoredb.py

```diff
@@ -60,6 +60,13 @@ def should_install(self, host_target):
     def install(self, host_target):
         pass
 
+    def supports_cross_compilation(self):
+        """
+        Whether the product can be built for a different architecture than the 
+        host by passing --cross-compile-host to it.
```

Comment:
Why not just override has_cross_compile_hosts on IndexStoreDB?

Reply:
Because I didn’t think of that. Good idea 👍